lib/coinbase/address/external_address.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Coinbase
+  # A representation of a blockchain Address, which is a user-controlled account on a Network. Addresses are used to
+  # send and receive Assets. An ExternalAddress is an Address that is not controlled by the developer, but is instead
```

Comment:
> An ExternalAddress is an Address that is not controlled by the developer, but is instead

This doesnt need to be true in all scenarios. ExternalAddress is an address that does not belong to a CDP Wallet